### DISCUSSION TITLE
zigbee2tasmota and zigbee2mqtt 2in 1 Gateway

### DISCUSSION THREAD

**Discussion by yzg011** [arendst/Tasmota#11268] [Show and tell]
1.  support zigbee2tasmota
2. support zigbee2mqtt
3. support ccloader update zigbee FW
4. support web update zigbee FW
5. support TCP sever default port 8880
6. support to keep zigbee FW in littlefs

![1edfa4ad9e483f4008d444d2ce6f0b6f71fdee61_2_375x500](https://user-images.githubusercontent.com/24872854/110416828-58b87000-80cf-11eb-8c1c-39f0cf97af90.jpg)
![0cdf85b985fbdc77427ecd55319252fef4ef4622_2_690x407](https://user-images.githubusercontent.com/24872854/110416838-5bb36080-80cf-11eb-9a13-1d7cf509a60e.png)
![ab40ec36d967ca5f0dfb205ac03b063](https://user-images.githubusercontent.com/24872854/110416845-5e15ba80-80cf-11eb-93a6-4852f57fd23b.png)

**Comment by nagyrobi** [comment]
What MCU inside?

**Reply by yzg011** [reply]
esp8266  and cc2530

**Reply by s-hadinger** [reply]
Do you have any plans for esp32 + cc2652?

**Reply by yzg011** [reply]
will do
